src/renderer/Voice.tsx

```diff
@@ -1194,6 +1209,13 @@ const Voice: React.FC<VoiceProps> = function ({ t, error: initialError }: VoiceP
 		const tempTalking = { ...otherTalking };
 		let talkingUpdate = false;
 		for (const player of otherPlayers) {
+			// bug with TOH where all players are considered alive during the ejection screen. This is a workaround
+			if (checkIfPlayerWasDeadAtStartOfMeeting(player.ptr) && !player.isDead) {
+				player.isDead = true;
+			}
+			if (checkIfPlayerWasDeadAtStartOfMeeting(myPlayer.ptr) && !myPlayer.isDead) {
+				myPlayer.isDead = true;
+			}
```

Comment:
Could be refactored out of the loop as we don't need to check the local players dead state for every other player.